ci-operator/step-registry/redhat-appstudio/e2e/redhat-appstudio-e2e-ref.yaml

```diff
@@ -16,6 +16,8 @@ ref:
     name: "release:latest"
   - env: CI_JBS_REQPROCESSOR_IMAGE
     name: "release:latest"
+  - env: CI_SPI_OAUTH_IMAGE
+    name: "release:latest"
```

Comment:
why not redhat-appstudio-spi-oauth-image? and also i dont understand why CI_JBS_REQPROCESSOR_IMAGE

Reply:
[explanation](https://github.com/redhat-appstudio/e2e-tests/blob/dd757fc91f25eef1daf4d453785424a3fec60670/magefiles/magefile.go#L332-L337)

Reply:
💥 head